docs/manual/host-configuration/index.md

```diff
@@ -62,14 +62,14 @@ If you have changed the *base\_url*, *base\_port* or *base\_dir* setting in */et
     bin_dir = /usr/lib/airtime/api_clients
     api_key = 'XXXXXXXXXXXXXXXXXXXX'
     api_base = api
-    host = airtime.example.com
+    host = libretime.example.com
     base_port = 80
     base_dir = /
 
 Media monitor settings
 ----------------------
 
-Airtime's media monitor has its own configuration file in */etc/airtime/media-monitor.cfg* which contains RabbitMQ settings and other preferences.
+Libretime's media monitor has its own configuration file in */etc/airtime/media-monitor.cfg* which contains RabbitMQ settings and other preferences.
```

Comment:
The media monitor has been deprecated so this section can be removed

Reply:
Will be in next PR.